h2o-algos/src/main/java/hex/schemas/GLMV3.java

```diff
@@ -155,6 +157,9 @@ public static final class GLMParametersV3 extends ModelParametersSchemaV3<GLMPar
     @API(help = "Standardize numeric columns to have zero mean and unit variance", level = Level.critical, gridable = true)
     public boolean standardize;
 
+    @API(help = "Internal parameter, do not use.  Have no effect on model.", level = Level.critical, gridable = false)
+    public boolean expose_constraints;
```

Comment:
I don't think you should expose internal parameters through the rest API.

Reply:
You are so right .  I even write it down in the comments!  Removed.